README.md

```diff
@@ -29,16 +29,13 @@ Before diving into the CLI, we recommend taking a look at the Rosetta API Docs:
 * [Construction API](https://www.rosetta-api.org/docs/construction_api_introduction.html)
 
 ## Install
-To install `rosetta-cli` from source, run:
-```
-go get github.com/coinbase/rosetta-cli
-```
-
 To download a binary directly for the latest release, run:
 ```
 curl -sSfL https://raw.githubusercontent.com/coinbase/rosetta-cli/master/scripts/install.sh | sh -s
 ```
 
+The binary will be installed inside the`./bin` directory from where the install command was run.
```

Comment:
nit: The binary will be installed inside the`./bin` directory relative to where the install command was run.